sorc/ncep_post.fd/IFI.F

```diff
@@ -792,7 +792,7 @@ END SUBROUTINE EXCH_c_float
 !$OMP PARALLEL DO COLLAPSE(2)
     do j=jsta,jend
        do i=ista,iend
-          if(ifi_apcp(i,j)>9e9 .or. ifi_apcp(i,j)<-9e9 .or. ifi_apcp(i,j)==0) then
+          if(ifi_apcp(i,j) == spval) then
```

Comment:
@SamuelTrahanNOAA A change log is needed at the beginning session of this subroutine.

Reply:
I added one. How do you like it?

Reply:
Hold on... I haven't pushed it yet.

Reply:
I've pushed a change log to the top of this file. How do you like it?